pages/common/glab.md

```diff
@@ -0,0 +1,28 @@
+# glab
+
+> GitLab CLI tool to help working with GitLab from the command line.
+> More information: <https://clementsam.tech/glab/>.
+
+- Create a merge request:
+
+`glab mr create`
+
+- List merge requests:
+
+`glab mr list`
+
+- Create a new issue:
+
+`glab issue create`
+
+- View and filter repository’s open issues:
```

Comment:
Perhaps this will read better?
```suggestion
- View and filter the current repository's open issues:
```

Also, we prefer regular ASCII here :-) I can't remember which issue it was that we discussed it, but I can dig it up if you're interested.